doc/source/index.rst

```diff
@@ -95,9 +95,12 @@ the base Satpy installation.
     * - MTSAT-2 Imager data in JMA HRIT format
       - `mtsat2-imager_hrit`
       - Beta
-    * - GOES 16 imager data in netcdf format
+    * - GOES-R imager data in netcdf format
```

Comment:
Since you added NOAA to the L2 reader, should it be added here too?

Reply:
The reason I add NOAA to the l2 reader is that I am not sure if there is any other org will share their l2 products too. just want to make it clear it works only with NOAA l2+ products. For the l1b, I think NOAA is the only data provider.

Reply:
Sounds good.